packages/runtime/datastore/package.json

```diff
@@ -102,6 +102,8 @@
   },
   "typeValidation": {
     "version": "2.0.0",
-    "broken": {}
+    "broken": {
+      "ClassDeclaration_FluidDataStoreRuntime": {"forwardCompat": false}
```

Comment:
is this necessary?

Reply:
I was also a bit confused here, but it seems to be. The new interface implemented by FDSR is optional (Partial<>) but the validatePrevious tests complain about the new IFluidLoadable property. Maybe because it's a getter, not a property with `?`? Let me see if I can do something there.